internal/confmapprovider/discovery/README.md

```diff
@@ -101,4 +101,50 @@ successfully started observers.
 1. Stop all temporary components before continuing on to the actual Collector service (or exiting early with `--dry-run`).
 
 
-By default, the Discovery mode is provided with pre-made discovery config components in [`bundle.d`](./bundle/README.md).
\ No newline at end of file
+By default, the Discovery mode is provided with pre-made discovery config components in [`bundle.d`](./bundle/README.md).
+
+
+### Discovery properties
+
+Configuring discovery components is performed by merging discovery properties with the config.d receivers
+and extensions `*.discovery.yaml` files. Discovery properties are of the form:
+
+```yaml
+splunk.discovery.receivers.<receiver-type(/name)>.config.<field>(<::subfield>)*: <value>
+splunk.discovery.extensions.<observer-type(/name)>.config.<field>(<::subfield>)*: <value>
+splunk.discovery.receivers.<receiver-type(/name)>.enabled.: <true or false>
+splunk.discovery.extensions.<observer-type(/name)>.enabled: <true or false>
+
+# Examples
+splunk.discovery.receivers.prometheus_simple.config.labels::my_label: my_label_value
+splunk.discovery.receivers.prometheus_simple.enabled: true
+
+splunk.discovery.extensions.docker_observer.config.endpoint: tcp://localhost:8080
+splunk.discovery.extensions.k8s_observer.enabled: false
+```
+
+These properties can be in `config.d/properties.discovery.yaml` or specified at run time with `--set` command line options.
+
+Each discovery property also has an equivalent environment variable form using `_x<hex pair>_` encoded delimiters for
```

Comment:
Is this format `_x<hex pair for the given delimeter>_` some kind of encoding standard? I'm not sure I've seen this.

Reply:
https://github.com/signalfx/splunk-otel-collector/pull/2494#discussion_r1083085874

I'm not familiar with a word character charset that I could use w/ idn or stringprep so defined the "wordify" encoding.

Reply:
That's fair, thanks for the reference!